neat_core.c

```diff
@@ -1078,7 +1078,9 @@ neat_change_timeout(neat_ctx *mgr, neat_flow *flow, int seconds)
         }
 
         return NEAT_ERROR_OK;
-    } else if (flow->sockProtocol == IPPROTO_SCTP) {
+    }
+#if defined(IPPROTO_SCTP)
```

Comment:
Then I would suggest to remove the
```
#if defined(IPPROTO_SCTP)
```
line. The same applies to the other places where the code checks if `IPPROTO_SCTP` is defined.